ajax/libs/valine/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "valine",
+  "filename": "Valine.min.js",
+  "description": "A simple comment system based on Leancloud.",
+  "version": "1.1.8",
+  "keywords": [
+    "Valine",
+    "Simple Comment",
+    "Comment System",
+    "Based on Leancloud",
+    "Minimalist"
+  ],
+  "author": "xCss <xioveliu@gmail.com> (https://github.com/xCss)",
+  "license": "GPL-2.0",
```

Comment:
Would you please discuss the lib under which license with the author?  GPL-2.0 or [ISC](https://github.com/xCss/Valine/blob/master/package.json#L20)
Here is a sample: https://github.com/davist11/jQuery-One-Page-Nav/issues/159
Many thanks.